src/main/kotlin/se/uulm/snowballr/backend/service/UserService.kt

```diff
@@ -150,16 +161,26 @@ class UserService(
         return targetUser.toGrpcUser()
     }
 
-    override suspend fun getAllUsers(): UserOuterClass.User.List {
+    override suspend fun getAllUsers(): GrpcUser.List {
         val currentUser = userRepo.getUserById(GrpcContext.getUserIdFromContext())
 
         verifyServerAdminRole(currentUser) { UnauthorizedException.All(EntityType.USER, AccessType.READ, it) }
 
         val users = userRepo.getAllUsers()
 
-        val builder = UserOuterClass.User.List.newBuilder()
-        users.forEach { builder.addUsers(it.toGrpcUser()) }
-        return builder.build()
+        return users.toGrpcUsers()
+    }
+
+    override suspend fun getInviteCandidates(request: GrpcUser.SearchQuery): GrpcUser.List {
+        val searchQuery = request.query.trim()
+
+        // Check whether the search query is too short, i.e., 3 or fewer characters long
+        if (searchQuery.length < MINIMUM_LENGTH_OF_SEARCH_QUERY) {
+            return GrpcUser.List.newBuilder().build()
```

Comment:
No must have, just a suggestion:
```suggestion
            return GrpcUser.List.getDefaultInstance()
```

Reply:
Did not know this, thank you for the hint.